src/orca-loader/Hyper.res

```diff
@@ -318,6 +318,14 @@ let make = (publishableKey, options: option<Js.Json.t>, analyticsInfo: option<Js
           addSmartEventListener("message", handleMessage, "onSubmit")
         })
       }
+
+      let oneClickConfirmPayment = (result: bool) =>
+        iframeRef.contents->Js.Array2.forEach(iframe => {
```

Comment:
Wrapping it in a promise and returning the confirm object will be required for this method.